packages/wonder-blocks-switch/package.json

```diff
@@ -19,7 +19,8 @@
     "@khanacademy/wonder-blocks-color": "^2.0.1",
     "@khanacademy/wonder-blocks-core": "^6.1.1",
     "@khanacademy/wonder-blocks-icon": "^2.1.5",
-    "@khanacademy/wonder-blocks-spacing": "^4.0.1"
+    "@khanacademy/wonder-blocks-spacing": "^4.0.1",
```

Comment:
**suggestion:** By including `theming` here, I think you can get rid of `color` and `spacing` as direct dependencies to this package.

If you still need to use them for Storybook or tests, you can move them to `devDependencies`.... although I think Storybook/tests can use `{tokens} from `wb-theming` instead so we can start introducing tokens gradually in the examples.